python/pyspark/sql/connect/README.md

```diff
@@ -49,3 +49,7 @@ To use the release version of Spark Connect:
 ./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic'
 ```
 
+## Generate proto generated files for the Python client
+1. Install `buf`: https://docs.buf.build/installation
```

Comment:
I think we also need to mention the related python packages: 

grpcio==1.48.1
protobuf==4.21.6
mypy-protobuf==3.3.0

Reply:
`buf`'s version in CI is `1.8.0`, the latest on is `1.9.0`.  I think we also need to mention the version

Reply:
makes sense. I updated this doc with versions.